src/emoji-row.jsx

```diff
@@ -25,14 +25,8 @@ export default class EmojiRow extends Component {
               ariaLabel={emoji.name}
               role="option"
               key={emoji.unicode}
-              onClick={function(e) {
-                onChange(emoji);
-              }}
-              onKeyUp={function(e) {
-                e.preventDefault();
-                if (e.which === 13 || e.which === 32) {
-                  onChange(emoji);
-                }
+              onSelect={() => {
```

Comment:
You can also do `onSelect={this.props.onChange}` and keep the Emoji component the responsability to  send the right information.

The difference will be that instead of creating a function for each emoji for each render, you'll be creating one function for each emoji only.